extractor/bug_info_test.go

```diff
@@ -47,32 +47,34 @@ func TestValidateCommentBody(t *testing.T) {
-	_, errMaps := ParseCommentBody(test)
+
+func TestParse(t *testing.T) {
+	var tmp = "## Please edit this comment to complete the following information\r\n\r\n### Not a bug\r\n\r\n1. Remove the 'type/bug' label\r\n2. Add notes to indicate why it is not a bug\r\n\r\n### Duplicate bug\r\n\r\n1. Add the 'type/duplicate' label\r\n2. Add the link to the original bug\r\n\r\n### Bug\r\n\r\nNote: Make Sure that 'component', and 'severity' labels are added\r\nExample for how to fill out the template: https://github.com/pingcap/tidb/issues/20100\r\n\r\n#### 1. Root Cause Analysis (RCA) (optional) \r\n<!-- Write down the reason why this bug occurs -->\r\n\r\n#### 2. Symptom (optional)\r\n\r\n<!-- What will the user see when this bug occurs. The error message may be in the terminal, log or monitoring -->\r\n\r\n#### 3. All Trigger Conditions (optional)\r\n\r\n<!-- All the user scenarios that may trigger this bug -->\r\n\r\n#### 4. Workaround (optional)\r\n\r\n#### 5. Affected versions\r\n\r\n<!--\r\nIn the format of [start_version:end_version], multiple version ranges are\r\naccepted. If the bug only affects the unreleased version, please input:\r\n\"unreleased\". For example:\r\n\r\nNotes:\r\n  1. Do not use any white spaces in '[]'.\r\n  2. The range in '[]' is a closed interval\r\n  3. The version format is `v$Major.$Minor.$Patch`, the $Majoy and $Minor\r\n     number in a version range should be the same. [v3.0.1:v3.1.2] is\r\n     invalid because the $Minor number of the version range is different.\r\n\r\nExample 1: [v3.0.1:v3.0.5], [v4.0.1:v4.0.5]\r\nExample 2: unreleased\r\n-->\r\n[v3.0.0:v3.0.19]\r\n#### 6. Fixed versions\r\nv3.0.20\r\n<!--\r\nThe first released version that contains this fix in each minor version. If the bug's affected version has been released, the fixed version should be a detailed version number; If the bug doesn't affect any released version, the fixed version can be \"master\";  \r\n\r\nExample 1: v3.0.13, v4.0.5\r\nExample 2: master\r\n-->"
```

Comment:
I think it might be writen in backticks.
https://blog.csdn.net/K346K346/article/details/105559800#t2

Reply:
Do you want it to be multi-lined so that it's easier to read?

Reply:
Yes

Reply:
There are some backticks in template, and it's no possible to escape backtick in raw string.
Handling this can make it look confusing.

Reply:
![image](https://user-images.githubusercontent.com/32479514/98323533-f23e1b00-2024-11eb-9145-d56aa9815202.png)

Reply:
It could work just like above.

Reply:
cool, fixed